fuse_models/test/test_unicycle_2d.cpp

```diff
@@ -5,6 +5,10 @@
  ***************************************************************************/
 #include <fuse_models/unicycle_2d.h>
 
+#include <string>
+#include <vector>
+
```

Comment:
Not sure, but I think more general headers are usually at the end in the `fuse` repo.